tests/test_setup_config_line_coverage.py

```
@@ -0,0 +1,210 @@
+    mocker.patch("fetchtastic.setup_config._crontab_available", return_value=True)
+    mocker.patch(
+        "fetchtastic.setup_config.get_fetchtastic_installation_method",
+        return_value="pip",
+    )
+    mocker.patch("fetchtastic.setup_config.config_exists", return_value=(False, ""))
+    mocker.patch("fetchtastic.setup_config.install_termux_packages")
+    mocker.patch("fetchtastic.setup_config.check_storage_setup", return_value=True)
+
+    mocker.patch(
+        "fetchtastic.setup_config._safe_input",
+        side_effect=["y", "/tmp/test"],
```

Ruff (0.15.12)

[error] Probable insecure usage of temporary file or directory: "/tmp/test"

(S108)